kubernetes/istio_rules_service.go

```diff
@@ -111,10 +111,15 @@ func (in *IstioClient) getActionHandler(namespace string, handlerType string, ha
 		return
 	}
 	result, err := in.istio.Get().Namespace(namespace).Resource(handlerTypePlural).SubResource(handlerName).Do().Get()
+	log.Infof("TODELETE %s/%s/%s", handlerTypePlural, handlerType, handlerName)
+	log.Infof("TODELETE %v", result)
+	log.Infof("TODELETE %v", err)
```

Comment:
I guess those logs are not intended be here, right?

Reply:
yeah, I completely forgot to clean them.
Good catch